app/views/users/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
You don't need to generate all the views, devise comes bundled with them you can use them without this folder.
You only need to generate the one that you want to change!
Delete this please

Reply:
Done